graphql-dgs-client/src/test/kotlin/com/netflix/graphql/dgs/client/ReactiveWebClientTest.kt

```diff
@@ -43,7 +43,7 @@ class ReactiveWebClientTest {
             .headers { consumer -> headers.forEach { consumer.addAll(it.key, it.value) } }
             .bodyValue(body)
             .exchange()
-            .flatMap { cr -> cr.bodyToMono(String::class.java).map { json -> HttpResponse(cr.rawStatusCode(), json) } }
```

Comment:
Did this change?

Reply:
The `rawStatusCode` doesn't exist anymore in Spring.